core/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitManager.java

```diff
@@ -90,7 +90,11 @@ public TwoPhaseConsensusCommitManager(DatabaseConfig databaseConfig) {
     coordinator = new Coordinator(storage, config);
     parallelExecutor = new ParallelExecutor(config);
     recovery = new RecoveryHandler(storage, coordinator, tableMetadataManager);
-    commit = new CommitHandler(storage, coordinator, tableMetadataManager, parallelExecutor);
+    // Create CommitHandler with throwExceptionIfCommittedTransactionExists false different from the
+    // case for ConsensusCommitManager because each service tries to put the state 'committed' in
```

Comment:
```suggestion
    // case for ConsensusCommitManager, because each service could put the state 'committed' in
```

Reply:
Thanks! According to Grammarly, the comma is not necessary, and I chose `can put` since we can still do it regardless of this PR. PTAL at b433e33.